src/components/sbb-teaser-hero/sbb-teaser-hero.tsx

```diff
@@ -109,15 +109,15 @@ export class SbbTeaserHero {
     return (
       <a class="teaser-hero" href={this.link} {...linkAttributes}>
         <sbb-image
+          aspect-ratio="1-1"
```

Comment:
why is aspect ratio set to 1-1 even when variant-teaser-hero should overwrite aspect ratio?

Reply:
aspect ratio can be anything... but 1-1 on small on variant-teaser-hero

Reply:
right